Automattic/woocommerce-payments
# Fix/8026 deposits list schedule consistent

Fixes #8026

#### Changes proposed in this Pull Request
This PR:

- Moves the https://github.com/Automattic/woocommerce-payments/issues/8175 into `DepositSchedule`.
- Adds a short-term css tweak to ensure the notice text is vertically centered.

But why?

- So the deposit schedule info shown in deposits list is consistent with payments overview.

<img width="1414" alt="Screenshot 2024-02-23 at 1 21 18 PM" src="https://github.com/Automattic/woocommerce-payments/assets/4167300/5c5bc5bd-fa82-41fa-a9f8-7cb4ce2618fc">

#### Testing instructions
* Ensure your store is onboarded & has some deposits.
* Go to `Payments > Deposits`.
* Ensure the notice is displayed (it's dismissable, so you may need to delete an option to un-dismiss it).
* Check that the notice has:
  * accurate info about deposit schedule
  * a tooltip to explain & link out to more info
  * no ETA or specific estimated date
  * generally should be consistent with `Payments > Overview`

Bonus points: check with a range of schedules, and store in new-account waiting period, deposits blocked etc.

-------------------


**Post merge**
